GithubHandler.py

```diff
@@ -129,7 +145,7 @@ def get_file_contents(self, filepath: str, ref=None):
             traceback.print_exc()
             return "Could not retrieve file contents."
 
-    def get_pr_deltas(self, pr):
+    def get_pr_deltas(self, pr, valid_extensions=None):
```

Comment:
The parameter `valid_extensions` is being passed around multiple methods, violating the DRY principle. It would be better to set a class-level default value that can be overridden if necessary.
        ```suggestion
        def get_pr_deltas(self, pr, valid_extensions=GPTsettings.FILE_EXTENSIONS):
        ```